web/source/keymanweb.js

```diff
@@ -1686,13 +1719,6 @@ if(!window['tavultesoft']['keymanweb']['initialized']) {
 
       // Update the UI 
       keymanweb.doKeyboardRegistered(sp['KI'],sp['KL'],sp['KN'],sp['KLC']);
-
-      // If the first stub, must load (and optionally display) the keyboard
-      if(keymanweb._KeyboardStubs.length == 1 && document.readyState=='complete') {
-        keymanweb._SetActiveKeyboard(sp['KI'], sp['KLC']);     
-        keymanweb.globalKeyboard = sp['KI'];
-        keymanweb.globalLanguageCode = sp['KLC'];
```

Comment:
Triggering the load of a keyboard from its stub here triggered the system to load the stub, not caring if a later part of initialization would set a different default.  (Possibly from cookie settings)

This is half of what broke the help.keyman.com example mentioned in the original PR comment.

Reply:
So cookie-based settings (e.g. for keymanweb.com) still work with this change?

Reply:
Yes.  This particular removal here prevents the first added stub from being loaded, which is an entirely separate section from where the cookie settings are loaded.  That part happens at the end of 'native' path `init`.